Sources/Unbox.swift

```diff
@@ -107,6 +107,30 @@ public func unbox<T: UnboxableWithContext>(data: Data, context: T.UnboxContext,
     return try data.unbox(context: context, allowInvalidElements: allowInvalidElements)
 }
 
+/// Unbox binary data into a dictionary of type `[String: T]`. Throws `UnboxError`.
+public func unbox<T: Unboxable>(data: Data) throws -> [String: T] {
+    let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: [String: Any]]
```

Comment:
There's an extension you can use here: `JSONSerialization.unbox()`, to get the same error handling code as the rest of the implementation.